rhc-ose-ansible/roles/registry/tasks/main.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install Registry
+  yum:
+    name: docker-registry
+    state: latest
+- name: Enable & Start Registry
+  service:
+    name: docker-registry
+    enabled: yes
+    state: started
+- name: Install firewalld
+  yum:
+    name: firewalld
+    state: latest
+- name: Enable Firewalld
+  service:
+    name: firewalld
+    enabled: yes
+    state: started
+- name: Open Firewall for Registry
+  firewalld:
```

Comment:
@etsauer having trouble accessing the registry from an external machine.

Adding `immediate: yes` to the firewalld module will reload the configuration and allow communication through the port